spec/rubocop/server/cache_spec.rb

```diff
@@ -241,4 +241,17 @@
       end
     end
   end
+
+  if RuboCop::Server.support_server?
```

Comment:
Can the test pass with `RuboCop::Platform.windows?`?

```suggestion
  if RuboCop::Platform.windows?
```

Reply:
I had an issue with the `project_dir_cache_key` being invalid on the windows CI, because a directory like `C:/foo/bar` was being sanitized as `:+foo+bar` which was throwing an exception, trying an approach where specific clean-up is performed for windows platform.

Will there be any plans to try to support Rubocop server for windows and is that worth the effort?

Reply:
Oops, that's my mistake. It's `unless RuboCop::Platform.windows?` instead of `if RuboCop::Platform.windows?`. It's okay if it doesn't work on Windows.

Reply:
Alright, I'd expect that to work without issue then